File: Payanak/Original/ClientApp/src/app/shared/model/sms/schedule-sms-info.model.ts
import {ContactModel} from '../contact/contact.model';
import {TemplateModel} from './template.model';
import {NumberModel} from '../number/number.model';

export class ScheduleSmsInfoModel {
  id: number;
  name: string;
  code: number;
  addedYear: number;
  addedMonth: number;
  addedDay: number;
  owner: ContactModel;
  template: TemplateModel;
  number: NumberModel;
  status: number;
}

File: Payanak/Original/ClientModels/ScheduleSmsInfoModel.cs
namespace Backend.ClientModels
{
    public class ScheduleSmsInfoModel
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public long Code { get; set; }
        public int AddedYear { get; set; }
        public int AddedMonth { get; set; }
        public int AddedDay { get; set; }
        public ContactModel Owner { get; set; }
        public TemplateModel Template { get; set; }
        public NumberModel Number { get; set; }
        public short Status { get; set; }
    }
}